package.json

```diff
@@ -111,6 +110,10 @@
     "webpack-bundle-analyzer": "^4.7.0",
     "webpack-cli": "^5.0.0"
   },
+  "engines": {
+    "npm": ">= 6.0.0",
+    "node": ">= 16.18.1"
```

Comment:
shouldn't this be `>= 18.18.2`?

Reply:
Why do you say so?

Reply:
6d03e46d12ee3d37ea6de52d86e2bc5aa46436ee i set it to the version used in containers